src/ivc/fold_relaxed_plonk_instance_chip.rs

```diff
@@ -185,13 +186,134 @@ impl<C: CurveAffine> AssignedRelaxedPlonkInstance<C> {
             folded_X1,
         })
     }
+
+    pub fn absorb_into(&self, ro: &mut impl ROCircuitTrait<C::Base>)
+    where
+        <C as halo2curves::CurveAffine>::Base: ff::PrimeFieldBits + ff::FromUniformBytes<64>,
+    {
+        let Self {
+            folded_W,
+            folded_E,
+            folded_u,
+            folded_challenges,
+            folded_X0,
+            folded_X1,
+        } = self;
+
+        folded_W.iter().for_each(|W| {
+            let (x, y) = WrapValue::from_assigned_point(W);
+            ro.absorb_point(x, y);
+        });
+
+        let (x, y) = WrapValue::from_assigned_point(folded_E);
+        ro.absorb_point(x, y);
+
```

Comment:
To match result of off-circuit, the order of absorb_into needs to be the same. W->E->inst->u.

Reply:
Fixed